filesystem/iso9660.ksy

```diff
@@ -408,48 +456,12 @@ types:
         type: b1
       - id: creation
         type: b1
-      - id: creation_short
-        type: datetime_short
-        if: ( creation == true ) and ( long_form == false )
-      - id: creation_long
-        type: datetime_long
-        if: ( creation == true ) and ( long_form == true )
-      - id: modify_short
-        type: datetime_short
-        if: ( modify == true ) and ( long_form == false )
-      - id: modify_long
-        type: datetime_long
-        if: ( modify == true ) and ( long_form == true )
-      - id: access_short
-        type: datetime_short
-        if: ( access == true ) and ( long_form == false )
-      - id: access_long
-        type: datetime_long
-        if: ( access == true ) and ( long_form == true )
-      - id: attributes_short
-        type: datetime_short
-        if: ( attributes == true ) and ( long_form == false )
-      - id: attributes_long
-        type: datetime_long
-        if: ( attributes == true ) and ( long_form == true )
-      - id: backup_short
-        type: datetime_short
-        if: ( backup == true ) and ( long_form == false )
-      - id: backup_long
-        type: datetime_long
-        if: ( backup == true ) and ( long_form == true )
-      - id: expiration_short
-        type: datetime_short
-        if: ( expiration == true ) and ( long_form == false )
-      - id: expiration_long
-        type: datetime_long
-        if: ( expiration == true ) and ( long_form == true )
-      - id: effective_short
-        type: datetime_short
-        if: ( effective == true ) and ( long_form == false )
-      - id: effective_long
-        type: datetime_long
-        if: ( effective == true ) and ( long_form == true )
+      - id: datetime_short
+        type: rrip_tf_short
+        if: long_form == false
+      - id: datetime_long
+        type: rrip_tf_long
+        if: long_form == true
```

Comment:
I guess ```== true``` is not needed. ```x == false``` is ```not x```.
Use ```switch-on``` and ```cases``` instead of sequential ```if```s.

Reply:
Done